### DISCUSSION TITLE
What purpose does ffmpeg.dll serve?

### DISCUSSION THREAD

**Discussion by EspressoDS** [jgraph/drawio-desktop#1158] [Q&A]
Hi. I'm just wondering what purpose ffmpeg serves in the draw.io desktop application?

**Comment by michaelwingender** [comment]
This is an interesting question. I was using Desktop App version 20.8.10. After trying to upgrade to latest version my Virusscanner is complaining about ffmpeg.dll and does not allow me to start the app.
